src/main/java/pro/trevor/tankgame/rule/impl/version4/TankAttribute.java

```diff
@@ -0,0 +1,48 @@
+package pro.trevor.tankgame.rule.impl.version4;
+
+import pro.trevor.tankgame.state.board.unit.tank.IAttribute;
+
+public enum TankAttribute implements IAttribute {
+    DEAD(Boolean.class),
```

Comment:
Bounty is not an attribute and the toJson method of GenericTank does not set it explicitly.